sdk/ai/azure-ai-client/samples/agents/sample_agents_functions.py

```diff
@@ -0,0 +1,90 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+import os, time
+from azure.ai.client import AzureAIClient
+from azure.identity import DefaultAzureCredential
+from azure.ai.client.models import FunctionTool
+from user_functions import user_functions
+
+# Create an Azure AI Client from a connection string, copied from your AI Studio project.
+# At the moment, it should be in the format "<HostName>;<AzureSubscriptionId>;<ResourceGroup>;<HubName>"
+# Customer needs to login to Azure subscription via Azure CLI and set the environment variables
+
+connection_string = os.environ["AI_CLIENT_CONNECTION_STRING"]
+
+ai_client = AzureAIClient.from_connection_string(
+    credential=DefaultAzureCredential(),
+    connection=connection_string,
+    # logging_enable=True, # Optional. Remove this line if you don't want to show how to enable logging
```

Comment:
We don't need to show the logging_enable variable. The package README.md will have a section describing how to turn on logging.

Reply:
ok, only reason why it was there, is that I copied what was in other samples (at some point) to keep it look the same. Will remove